recipes/pvcaptest/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
It seems the license is already being packaged on pypi, you can remove this file then

Reply:
I removed the duplicate license file.